packages/utils/src/common/sharding.ts

```diff
@@ -177,3 +176,18 @@ export function contentTopicsByPubsubTopic(
   }
   return groupedContentTopics;
 }
+
+/**
+ * Used when creating encoders/decoders to determine which pubsub topic to use
+ */
+export function determinePubsubTopic(
+  contentTopic: string,
+  pubsubTopicShardInfo?: SingleShardInfo,
+  defaultPubsubTopic: string = "/waku/2/default-waku/proto"
```

Comment:
nit: let's refer to the constant here and not hardcoded value

Reply:
fixed in latest commit